pkg/linter/lib/src/lint_codes.g.dart

```diff
@@ -623,8 +623,9 @@ class LinterLintCode extends LintCode {
 
   static const LintCode do_not_use_environment = LinterLintCode(
     LintNames.do_not_use_environment,
-    "Invalid use of an environment declaration.",
-    correctionMessage: "Try removing the environment declaration usage.",
+    "Avoid using environment values like '{0}' which create hidden global state.",
```

Comment:
I don't understand what "hidden global state" means in this case.
Or how reading the compilation environment "create"s anything.